src/resources/elements/dropdown/dropdown.scss

```diff
@@ -34,6 +34,7 @@
     cursor: pointer;
     margin: 11px;
     margin-right: 0;
+    height: 21px;
```

Comment:
Why?  this makes me nervous because it has potential wide effect beyond what you are trying to achieve here.

Reply:
reverted in commit 3bb80d3. Instead I've changed the tokenInfo height. This is in order to keep the height from changing and dragging the whole view below up or down, in case there's multiple icon sizes.